README.md

```diff
@@ -1,24 +1,74 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget App
 
-Things you may want to cover:
+A Budget-app is mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+![screenshot](./screenshot-1.png)
 
-* System dependencies
 
-* Configuration
+## Live Version
 
-* Database creation
+[See Live Version](https://budgettransaction-app.herokuapp.com/)
```

Comment:
- [ ] The live demo is still not working. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/178095770-58998d37-6f0a-4cb0-992b-76b43df97c06.png)

Reply:
I have tried resolving the issue but it's still not working,can you please help